### ISSUE TITLE
[ACCESSIBILITY] Data Filters > search bar is missing a form label

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#542]
In the Explorer > select region Counties, School Districts, or Schools > go to Data Filters > the search bar in the panel is missing a form label. 

- What It Means: A form control does not have a corresponding label.
- Why It Matters: If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.
- What To Do: If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.
- 
![image](https://user-images.githubusercontent.com/76966798/106820187-c2a4ac00-662f-11eb-9222-87bc17779cbf.png)

**Comment by laurendoone** [comment]
retested dev but still seeing the error on the testing site: 
![image](https://user-images.githubusercontent.com/76966798/107099061-9327a800-67c5-11eb-9780-e8a4ee51840f.png)
